### PR TITLE
Get data users

### PR DESCRIPTION
**Pull request by at-tamnguyen4**
## Description
A few sentences describing the overall goals of the pull request's commits.
## Reference Requirements
- Product Backlog ID: PBI-1
- Task ID: TI-1

## Migrations
YES | NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: app/Http/Controllers/Admin/UserController.php [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 use Illuminate\Http\Request;
 use App\Http\Controllers\Controller;
+use Illuminate\Support\Facades\DB;
```
Comment (at-xinhnguyen) [inline app/Http/Controllers/Admin/UserController.php]: chỉ cần `use DB`; là được rồi

File: app/Http/Controllers/Admin/UserController.php [hunk 2]
```diff
@@ -14,7 +16,20 @@ class UserController extends Controller
      */
     public function index()
     {
-        return view('backend.users.index');
+        $field = [
+            'users.id',
+            'users.employee_code',
+            'users.name',
+            'users.email',
+            DB::raw('count(distinct(borrowings.id)) as total_borrowed'),
```
Comment (at-xinhnguyen) [inline app/Http/Controllers/Admin/UserController.php]: viết hoa các mysql keyword em nhé 
count -> COUNT, DISTINCT, AS

File: app/Http/Controllers/Admin/UserController.php [hunk 3]
```diff
@@ -14,7 +16,20 @@ class UserController extends Controller
      */
     public function index()
     {
-        return view('backend.users.index');
+        $field = [
```
Comment (at-xinhnguyen) [inline app/Http/Controllers/Admin/UserController.php]: $field có `s` nhé đây là nhiều trường

Comment (at-tamnguyen4) [reply app/Http/Controllers/Admin/UserController.php]: a xem giúp e mấy cái tên cột trong table có ổn chưa

**Pull request merged**